examples/src/main/python/ml/count_vectorizer_example.py

```diff
@@ -17,9 +17,10 @@
 
 from __future__ import print_function
 
-from pyspark.sql import SparkSession
 # $example on$
 from pyspark.ml.feature import CountVectorizer
+from pyspark.sql import SparkSession
```

Comment:
Similar comment as previous reordering of SparkSession import, consider moving $example off$ above this import so examples remain the same.